doozer/doozerlib/cli/images_konflux.py

```diff
@@ -382,7 +385,7 @@ async def _rebase_and_build(
                 logger.info(f"A previous bundle build already exists: {bundle_build.nvr}")
                 if not self.force:
                     logger.info("Skipping because --force is not set")
-                    return
+                    return bundle_build.nvr
```

Comment:
is there a test that can be updated here to make sure a string value is being returned from this func?